src/sbuf.c

```diff
@@ -938,56 +951,100 @@ static void setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 			tls_config_verify_client_optional(conf);
 		}
 	}
+
+	return true;
 }
 
-void sbuf_tls_setup(void)
+bool sbuf_tls_setup(void)
 {
 	int err;
+	bool ok;
+	struct tls_config *new_client_accept_conf = NULL;
+	struct tls_config *new_server_connect_conf = NULL;
```

Comment:
You said you are only dealing with `client_tls_*` settings in this patch. Why is this code here concerning itself with the server side of things then?

Reply:
I used the `tag_pool_dirty` function now to also allow changing of `server_tls_*` settings.